core/test/solver/ir.cpp

```diff
@@ -319,4 +319,24 @@ TYPED_TEST(Ir, ThrowOnWrongInnerSolverSet)
 }
 
 
+TYPED_TEST(Ir, UseAsRichardson)
```

Comment:
Can we test somewhere that the default value (1.0) is correct?

Reply:
Yes I add a DefaultRelaxationFactor to test the default is 1